lib/SILOptimizer/Mandatory/AddressLowering.cpp

```diff
@@ -1063,6 +1084,10 @@ void ValueStorageMap::recordComposingUseProjection(Operand *oper,
   if (userValue->getType().getEnumOrBoundGenericEnum()) {
     storage.initializesEnum = true;
   }
+  // FIXME: FIXME_NOW: Something like this...
+  if (userValue->getType().isExistentialType()) {
+    storage.initializesEnum = true;
```

Comment:
We should comment that `initializesEnum` is for the payload, not the enum. It maps to the instruction that initializes the payload, not the `enum` instruction.

Reply:
Fixed.